questions.md

```diff
@@ -4,26 +4,82 @@
 
 1. On the UCI cluster, the resource request "-pe openmp 64" refers to the number of processors requested.  Does that
    request mean that your commands will use multiple processors?
+   
+   __No. 64 processors will be allocated, but your command may still only use one core__
+   
 2. In general, how do you know how many processors, how much RAM, how many files would be required/needed/written by the
    jobs you are running on the cluster?
+   
+   __You can figure out how many processors are required, RAM needed and files written by the job by prefixing the program command you are running on the cluster with ```/usr/bin/time -v```. To estimate this before running a job to get an idea of the resources you should request, you can perform this same operation on a smaller set of input data than you would run in your actual job, and then scale the resource information it provides up (i.e. use "Percent of CPU this job got", "Maximum resident set size (kbytes)", and "File system outputs" from the run on a smaller set of data to estimate processsor, RAM, and file requirements needed when the real job is run.__
+   
```

Comment:
Very good answer. this is good but it is is also good to add option -o which allows an output file. env time -v -o MYOUTPUTFILE CMD would do the trick and do a dump to a text file. this helps sometimes if you care about what goes in your stdout and stderr files. I like to keep it seperate as I usually do tail -n 1 MYSTDOUTFILE to see if I was able to run to completion and then sort my STDERRFILES using ls -lS jobname.ejobid* by size to see if the stderr produced larger output than normal. this typically means there where errors on the run. This doesn't work though when the programmers are sloppy and lazy and output stdout to stderr. :( In the end you could also just check the finish error code in outputfile.